docs/modules/user-guide/partials/ref_devfile-samples.adoc

```diff
@@ -1,6 +1,7 @@
 [id="ref_devfile-samples_{context}"]
-= Devfile Samples
+= Devfile Resources
 
 .Additional Resources
 
-* https://github.com/devfile/api/tree/master/samples/devfiles[Devfile Samples]
+* To view devfile issues and open issues, go to https://github.com/devfile/api/tree/master/samples/devfiles[Devfile Samples]
```

Comment:
We probably should have 2 different entries:
1. Should the issue link be: https://github.com/devfile/api/issues?
2. add a different description for the sample link.